pkg/util/azureclient/applens/applens_client_options.go

```diff
@@ -26,7 +26,7 @@ func NewClientOptions() *ClientOptions {
 		azcore.ClientOptions{
 			Retry: policy.RetryOptions{
 				MaxRetries: 3,
-				RetryDelay: time.Second * 10,
+				RetryDelay: time.Second * 3,
```

Comment:
3 retries times 10 seconds.. that's 30 seconds. What's the timeout per try?

Reply:
It has an exponential back off, with a setting of 3 seconds it was erroring out in ~30 seconds (3 seconds + 12 seconds + round trip / response time from all 3 calls).  With a setting of 4 seconds, it was erroring out around 50 seconds.

To be safe I went with a setting of 3 seconds.

Reply:
That makes sense, but isn't immediately obvious. Can we add some code comments here as breadcrumbs for the next person that comes across this particular setting?

Reply:
Done

Reply:
Note on the place where it defined the setting it already calls out that its exponential.